src/masternodes/rpc_poolpair.cpp

```diff
@@ -601,6 +601,11 @@ UniValue createpoolpair(const JSONRPCRequest& request) {
         targetHeight = ::ChainActive().Height() + 1;
     }
 
+    const auto symbolLength = targetHeight >= Params().GetConsensus().GrandCentralHeight ? CToken::MAX_TOKEN_POOLPAIR_LENGTH : CToken::MAX_TOKEN_SYMBOL_LENGTH;
```

Comment:
The corresponding code on consensus side is based on the FortCanningHeight fork, please change GrandCentralHeight to FortCanningHeight.

https://github.com/DeFiCh/ain/blob/1dff273bfb948eb9ff378c0a5d36ed2f7c6a47f3/src/masternodes/mn_checks.cpp#L1180